spec/acceptance/basic_webhook_spec.rb

```diff
@@ -58,5 +58,17 @@ class {'r10k::webhook':
         expect(r.exit_code).to eq(0)
       end
     end
+
+    it 'should successfully lock when hammered with multiple requests' do
```

Comment:
Put the `it` tests inside the shell loop to remedy the rubocop warnings on too many expectations. The other rubocop warnings should be pretty explanatory except the array one, but I've not seen that before, sorry.